### PR TITLE
fix: don't create new file from palette

### PR DESCRIPTION
**Pull request by Montel**

### CODE REVIEW

File: src/core/project.h [hunk 1]
```diff
@@ -58,7 +58,7 @@ class Project : public QObject
 
 public slots:
     Core::Document *get(const QString &fileName);
-    Core::Document *open(const QString &fileName);
+    Core::Document *open(const QString &fileName, bool onlyExistingFile = false);
```
Comment (narnaud) [inline src/core/project.h:61]: I would prefer to not change this method, as it's an API.

File: src/gui/palette.cpp [hunk 2]
```diff
@@ -481,7 +481,7 @@ void Palette::addFileSelector()
         model->resetFileInfo();
     };
     auto selectFile = [](const QVariant &path) {
-        Core::Project::instance()->open(path.toString());
+        Core::Project::instance()->open(path.toString(), true);
```
Comment (narnaud) [inline src/gui/palette.cpp:484]: What you could do is query the `FileModel` to check if the file exists here, and only call open if it does.

Comment (Montel) [reply src/gui/palette.cpp:484]: ok I will do it.

**Comment by narnaud** [comment]
Close, please have a look at #216

**Pull request closed without merging**